src/components/ha-form/ha-form-integer.ts

```diff
@@ -61,6 +61,16 @@ export class HaFormInteger extends LitElement implements HaFormElement {
                 .disabled=${this.data === undefined}
                 @value-changed=${this._valueChanged}
               ></ha-paper-slider>
+              <paper-input
```

Comment:
This change is not reflected in PR description. Is this on purpose  ?

Reply:
Oops, forgot to take that out...